code/game/objects/items/implants/implant_circuit.dm

```diff
@@ -0,0 +1,51 @@
+/obj/item/implant/integrated_circuit
+	name = "electronic implant"
+	desc = "It's a case, for building tiny-sized, implantable electronics with."
+	icon = 'icons/obj/assemblies/electronic_setups.dmi'
+	icon_state = "setup_implant"
+	actions_types = list()
+	item_flags = null
+	allow_multiple = TRUE
```

Comment:
uhh. How is this balanced.

Reply:
Due to many parts, like most manipulators, not fitting in the tiny assemblies. Also, it's got like 32 complexity/12 space cap per implant. I think the worst thing one can expect here is people stacking stuff like an implantable flashlights and maybe some chem auto-injectors or something similar, although, it's not like a lot of this stuff can't be done with nanites.

Reply:
Nanites being used as a comparison for why something should be added without even bothering to consider how something will be abused is why TG had nanites "replacing" circuitry after circuitry was nerfed.

Reply:
Also, it's a straight port from the vorestation codebase. The implants can be stacked there just fine.

Reply:
that doesn't mean their implimentation is in any way, shape, or form balanced. Vorestation's an ERP group with a game added to it, not an active SS13 server

Reply:
I'm still not entirely seeing the power here, at least in comparison to say... carrying a bunch of circuits with you the traditional way. Especially since you miss out on a lot of functionalities once the implant is in.

Reply:
Getting inter-circuit communication working would make it fairly tedious to actually set up a whole lot of complex stuff with implantable circuits without relying on an external server circuit to handle all of the input/output